src/modules/previewpane/SvgPreviewHandler/Utilities/SvgPreviewHandlerHelper.cs

```diff
@@ -47,6 +47,7 @@ public static bool CheckBlockedElements(string svgData)
                 var elements = doc.Descendants().ToList();
                 foreach (XElement element in elements)
                 {
+                    // Using CurrentCulture since this is user facing
```

Comment:
These are element names in the SVG file, and we are checking if it matches any of `script`, `image`, or `feimage`. Might be a case for Ordinal/Invariant here? I'm not as familiar with the SVG file format, but @ryanbodrug-microsoft and @somil55 might have context here.

Reply:
This should be ordinal as we are doing an exact match for HTML tags

Reply:
Thanks @arjunbalgovind @somil55. I will change this to Invariant since CultureInfo does not have an option for Ordinal

Reply:
Good catch.  I believe this should be ToLowerInavriant if we want this to behave the same in every culture.